syntax="proto2";

package aap_protobuf.service.mediaplayback;

enum MediaPlaybackStatusMessageId
{
    MEDIA_PLAYBACK_STATUS = 32769;
    MEDIA_PLAYBACK_INPUT = 32770;
    MEDIA_PLAYBACK_METADATA = 32771;
}
